handlers/s3client.go

```diff
@@ -48,3 +51,28 @@ func StreamObjectFromS3(client *AwsClient, bucket string, key string) ([]byte, e
 	return buff.Bytes(), nil
 
 }
+
+// GetObjectSize returns object size in bytes
+func GetObjectSize(client *AwsClient, bucket string, key string) (*int64, error) {
```

Comment:
what is the point of the `*int64` type? does the `nil` value represent the object doesn't exist?

Reply:
I am using AWS s3 HeadObject object containing a pointer to the size of an object. If the object does not exist, the pointer is nil